### ISSUE TITLE
Request fo Boxnovel.com

### ISSUE THREAD

**Issue by Rajvir8** [JimmXinu/FanFicFare#440]
Apparently the previous site got blocked and Gamers of the Underworld basically lost it's place.

I checked and Boxnovel is also unavailable in the supported website list so I thought I'd request it as well.

An example is
https://boxnovel.com/novel/gamers-of-the-underworld/

**Comment by JimmXinu** [comment]
FYI:  I'm personally not interested in adding new sites these days.  Nor have other developers been very active lately.

I'm also not convinced that all of these translation sites should be in FanFicFare at all.  I haven't made a detailed study of the legalities, but I worry about the copyright issues of distributing translations of works from other languages.  Looking into it again briefly, it looks like it's mostly a matter of the Chinese/other source copyright holders not knowing or not caring about enforcement.

That said, I have [rejected](https://github.com/JimmXinu/FanFicFare/issues/221) [some](https://github.com/JimmXinu/FanFicFare/issues/223) [sites](https://github.com/JimmXinu/FanFicFare/issues/359) before for hosting content I'm reasonably sure should be considered copyrighted in my country, at least.

If someone wants to contribute code to add a new site, I'm usually happy to accept it.  But I reserve the right to reject (or even remove after inclusion) support for sites that might cause me problems.

BTW, wuxiaworld.site(#439) and boxnovel.com(#440) appear to be near-clones of each other.

**Comment by Rajvir8** [comment]
Interesting, and that's definitely fair.

I didn't really consider the legality of adding support and I care enough about Fanfic Fare that I wouldn't want it to be shut down like that.

Should I close the issue then, or leave it up in case the other Developers ever feel like adding more new sites?

**Comment by JimmXinu** [comment]
I'd leave it open for now.  More likely for the next person coming in to see and read the discussion that way.

**Comment by rdragonrydr** [comment]
I'll add that I can understand the whole "of questionable legality" thing, but I mainly use this to archive things I can read online (free access) anyway. It's more of a time-shifting (the thing that VCRs were sold as, but no one actually did) purpose, and at least Boxnovel is not pay-per-chapter. That's my take on it.

I was hoping when I read this that it would say that it was in progress. :grin:

**Comment by JimmXinu** [comment]
The people who win lawsuits are the people who can pay lawyers.  And in many cases, who can pay for lawyers the longest.

If there weren't companies behind the VCR with pockets deep enough to fund lawyers all the way to the SCUS, I'm confident it would not have shaken out the way it did.

And just because you and I use FFF for our personal libraries doesn't stop other people.  Like the guy who downloaded all of ffnet and shared it.

**Comment by rdragonrydr** [comment]
Okay, that's a very good point. And I have no idea what to even say about the latter. I have so many questions now...

**Comment by NightMachinary** [comment]
@rdragonrydr I use https://github.com/dipu-bd/lightnovel-crawler via its Telegram UI for translated novels (usually via `novelfull.com`), and it's great, except it's too interactive, and not scriptable.

Considering `youtube-dl` was reinstated, I highly doubt that creating a tool to scrape these sites is illegal. It's like saying a torrent downloader is illegal. I think the worst-case scenario is to just delete the code after receiving complaints.

**Comment by rdragonrydr** [comment]
> Considering `youtube-dl` was reinstated, I highly doubt that creating a tool to scrape these sites is illegal. It's like saying a torrent downloader is illegal. I think the worst-case scenario is to just delete the code after receiving complaints.

True, but I would rather not get this project deleted. Even temporarily. I would also like to point out that YTDL was only reinstated after the entire FSF hogpiled whoever was behind the takedown with the (seemingly obvious) fact that the claim was ridiculous. This has less support and is probably harder to justify or prove that a huge number of people need it.

Even if I too wish support existed.

**Comment by stranger-danger-zamu** [comment]
Just bumping with information.

Boxnovel (and all the sites that appear to be very similar) are just WordPress utilizing Madara a WordPress theme made explicitly for manga and which I assume has the ability to host text posts as well.

**Comment by JimmXinu** [comment]
boxnovel.com redirected to bronovel.com, which tells you to go to novgo.co.  Which I'm still not going to add.
